spec/googleauth/apply_auth_examples.rb

```diff
@@ -17,6 +17,7 @@
 $LOAD_PATH.uniq!
 
 require "faraday"
+require "google-cloud-env"
```

Comment:
Use `require "google/cloud/env"`. Requiring the gem name is a bundler convention but isn't generally done in normal code.